secure_message/api_mocks/internal_user_service_mock.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from structlog import wrap_logger
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+class InternalUserServiceMock:
+    @staticmethod
+    def get_user_details(uuid):
+        """gets the user details from the internal user service"""
+        logger.debug("getting mock user details for uaa")
+        if uuid:
+            internal_user_dict = {"id": uuid,
+                                  "firstName": "fred",
+                                  "lastName": "flinstone",
+                                  "emailAddress": "mock@email.com"}
+            return internal_user_dict, 200
```

Comment:
Do you need to assign `internal_user_dict` before returning it or could you return it directly without the assignment?

Reply:
This needs to accept a uuid and select the relevant dictionary entry . I think those tests we discussed yesterday will help ?